app/controllers/admin/upload_problems_report_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Admin::UploadProblemsReportController < ApplicationController
+  before_action :authenticate
+
+  def index
+    @cases_with_upload_problems = TribunalCase.with_upload_problems.order(created_at: :desc)
+  end
+
+  private
+
+  def authenticate
+    authenticate_or_request_with_http_basic do |_user, password|
+      Digest::SHA1.hexdigest(password) == ENV.fetch('UPLOAD_PROBLEMS_REPORT_AUTH_DIGEST')
```

Comment:
Is there any reason not to use sha256 here? I know it's overkill, but as a general rule I don't think we should be depending on sha1, these days.